client/state/domains/suggestions/actions.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Internal dependencies
+ */
+import wpcom from 'lib/wp';
+import {
+	DOMAINS_SUGGESTIONS_RECEIVE,
+	DOMAINS_SUGGESTIONS_REQUEST,
+	DOMAINS_SUGGESTIONS_REQUEST_FAILURE,
+	DOMAINS_SUGGESTIONS_REQUEST_SUCCESS
+} from 'state/action-types';
+
+/**
+ * Returns an action object to be used in signalling that a domains suggestion object
+ * has been received.
+ *
+ * @param   {Array}    suggestions              domain suggestions
+ * @param   {Object}   queryObject              domain suggestions queryObject
+ * @returns {Object}   Action object
+ */
+export function receiveDomainsSuggestions( suggestions, queryObject ) {
+	return {
+		type: DOMAINS_SUGGESTIONS_RECEIVE,
+		queryObject,
+		suggestions
```

Comment:
why we are not mapping the suggestions list creating a new domian object for each suggestion like in other instances?

```es6
{
  type: DOMAINS_SUGGESTIONS_RECEIVE,
  queryObject,
  suggestions: map( suggestions, createDomainSuggestionObject )
}
```

Trying to get something like ...

```es6
{
  domainName: "gwwar.me",
  const: "$25.00",
  productId: 46,
  productSlug: "dotme_domain"
}
```